kinect_position_x: -0.195116430521
kinect_position_y: 0.356311738491
kinect_position_z: 1.61279475689
kinect_orientation_x: 0.911406660555
kinect_orientation_y:  0.104320903672
kinect_orientation_z: -0.269073943437
kinect_orientation_w: -0.293350065821
